test/unit/cog_api/http/users_test.exs

```diff
@@ -50,4 +72,42 @@ defmodule CogApi.HTTP.UsersTest do
       end
     end
   end
+
+  describe "update" do
+    it "returns the updated user" do
+      cassette "users_update" do
+        params = %{
+          first_name: "Arnold",
+          last_name: "Vinick",
+          email_address: "arnold@example.com",
+          username: "arnie",
+          password: "12345",
+        }
+        endpoint = valid_endpoint
+        {:ok, user} = Client.user_create(endpoint, params)
+
+        params = %{first_name: "Arnie"}
+        {:ok, updated_user} = Client.user_update(endpoint, user.id, params)
+
+        assert updated_user.first_name == params.first_name
+      end
+    end
+  end
+
+  describe "delete" do
+    it "returns :ok" do
+      cassette "user_delete" do
+        endpoint = valid_endpoint
+        params = %{
+          first_name: "Josh",
+          last_name: "Lyman",
+          email_address: "josh@example.com",
+          username: "deputy_cos",
+          password: "password",
+        }
+        {:ok, user} = Client.user_create(endpoint, params)
+        assert :ok == Client.user_delete(endpoint, user.id)
```

Comment:
Same question as above. Should we also verify that it does not exist in the index here?